eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -18,26 +31,22 @@ jobs:
       Linux_x64:
         OSVmImage: 'ubuntu-18.04'
         vcpkg.deps: ''
-        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
         AZ_SDK_C_NO_SAMPLES: 'true'
       Win_x86:
         OSVmImage: 'windows-2019'
         vcpkg.deps: ''
-        VCPKG_DEFAULT_TRIPLET: 'x86-windows-static'
```

Comment:
We aren't using the triplet, even on windows? What do we do instead?

Reply:
We don't need triplets for vcpkg when there are no vcpkg dependencies. These triplets are redundant.